pkgs/development/python-modules/prometheus-pandas/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-zysdlYPjvhextAfW4vvGN6mSPVUbWLV+AvQ1hGWCAbw=";
+    };
+
+    propagatedBuildInputs = [
+      ipython
+      matplotlib
+      numpy
+      pandas
+      requests
+    ];
```

```suggestion
    propagatedBuildInputs = [
      numpy
      pandas
    ];
```
I think we need only numpy and pandas.
https://github.com/dcoles/prometheus-pandas/blob/89b3000f24de74a985411e907f50fbaed4141e47/setup.py#L21-L24

---

Nice catch, I'm not sure where I got the other ones from, must've been copy pasta 🙈.

---

Done.